src/intl/en/page-what-is-the-ethereum-network.json

```
@@ -0,0 +1,89 @@
+  "page-what-is-ethereum-network-staking-section-description-2": "When Ethereum started out, it used a consensus mechanism (a way to agree on who owns what) called proof-of-work. This is the same mechanism Bitcoin uses today.",
+  "page-what-is-ethereum-network-staking-section-description-3": "In September 2022, Ethereum upgraded to a more secure and energy-efficient proof-of-stake consensus mechanism.",
+  "page-what-is-ethereum-network-staking-section-description-4": "So, how does it work?",
+  "page-what-is-ethereum-network-staking-section-description-5": "In simple terms, <strong>people lock up (stake) some ETH</strong> as a deposit so that they can help <strong>secure the network</strong>. These people are called validators. When you stake ETH, your validator gets chosen to check and add new transactions. If you do it honestly, you <strong>earn rewards</strong>. If you <strong>try to cheat, you lose part of your stake</strong>.",
+  "page-what-is-ethereum-network-staking-section-description-6": "Staking is how Ethereum credibly commits to its service quality. All of this money at stake has the best interest for Ethereum to remain secure—would you bet against it?",
+  "page-what-is-ethereum-network-staking-section-description-7": "Just two years after launching proof-of-stake, Ethereum attracted <validators>over a million validators</validators> who stake <stakedEther>millions of ETH</stakedEther> to secure Ethereum. Together, they <strong>make it extremely expensive and difficult to attack</strong>. This is because, to attack the network, <strong>a bad actor needs at least 33 percent of all staked ETH</strong>. Today, that amounts to tens of billions of dollars, and even then, the attack would likely fail because of built-in safeguards.",
+  "page-what-is-ethereum-network-staking-section-description-8": "This is what gives Ethereum “<strong>economic security</strong>”. It's not just about having the right technology. It's about making attacks too costly to even try.",
+  "page-what-is-ethereum-network-staking-section-description-9": "To help secure the Ethereum network, you can do this in two main ways.",
+  "page-what-is-ethereum-network-staking-section-description-10": "The first way is running a node. Nodes store the entire history of the blockchain, including all transactions and smart contract data. By syncing with other nodes, they can agree on the state of the network, making sure transactions are legit and smart contract data is available.",
+  "page-what-is-ethereum-network-staking-section-description-11": "The second way is staking your ETH. The easiest way is through a staking provider like <lido>Lido</lido> or <rocketpool>Rocketpool</rocketpool> but you have the know how, try running validator software at home.",
```

Love placing pooled providers first ❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️

 ```suggestion
  "page-what-is-ethereum-network-staking-section-description-11": "The second way is staking your ETH. The easiest way is through a staking provider like <lido>Lido</lido> or <rocketpool>Rocketpool</rocketpool>. but if you have the know-how, try running validator software at home.",
```